account_bank_receipt_cancel_reason/model/account_bank_receipt.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+
+from openerp import models, fields
+
+
+class AccountBankReceipt(models.Model):
+    _inherit = 'account.bank.receipt'
+
+    cancel_reason_txt = fields.Char(
+        string="Description",
+        readonly=True)
```

Comment:
Use python coding standard, should be,

 cancel_reason_txt = fields.Char(
    string="Description",
    readonly=True,
)